pkg/print/v1pb.go

```
@@ -48,3 +49,94 @@ func CallerStatusResponse(w io.Writer, r *trustypb.CallerStatusResponse) {
 	table.Render()
 	fmt.Fprintln(w)
 }
+
+// Issuers prints list of IssuerInfo
+func Issuers(w io.Writer, issuers []*trustypb.IssuerInfo, withPem bool) {
+	now := time.Now()
+	for cnt, ci := range issuers {
+		fmt.Fprintf(w, "==================================== %d ====================================\n", cnt+1)
+		certBytes := []byte(ci.Certificate)
+		bundle, bundleStatus, err := certutil.VerifyBundleFromPEM(certBytes, []byte(ci.Intermediates), []byte(ci.Root))
+		if err != nil {
+			fmt.Fprintf(w, "ERROR: unable to create bundle: %s\n", err.Error())
+
+			crt, err := certutil.ParseFromPEM(certBytes)
+			if err == nil {
```

I wonder if ignoring the error in this case is OK?